fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/literal/DecimalV3Literal.java

```diff
@@ -84,6 +84,12 @@ public DecimalV3Literal roundFloor(int newScale) {
                 value.setScale(newScale, RoundingMode.FLOOR));
     }
 
+    public DecimalV3Literal round(int newScale) {
+        return new DecimalV3Literal(DecimalV3Type
+            .createDecimalV3Type(((DecimalV3Type) dataType).getPrecision(), newScale),
```

Comment:
what scale should be set if newScale bigger than old scale?

Reply:
when new scale is larger than original one, it would throw an Invocationexception when calling create Decimal Type in java with message: precision should not smaller than scale and ExpressionEvaluator would catch this exception and go back to original expression. But actually when new scalar bigger than old one, we can return original one when folding constant. Same as be calculation logic

Reply:
done

Reply:
> when new scale is larger than original one, it would throw an Invocationexception when calling create Decimal Type in java with message: precision should not smaller than scale and ExpressionEvaluator would catch this exception and go back to original expression. But actually when new scalar bigger than old one, we can return original one when folding constant. Same as be calculation logic

we should not rely on exception, exception is very expensive, so we should process it without use exception